tests/system/providers/google/workflows/example_workflows.py

```diff
@@ -182,21 +187,36 @@
         workflow_id=SLEEP_WORKFLOW_ID,
     )
 
+    cancel_execution_id = create_execution_for_cancel.output["execution_id"]
+
     # [START how_to_cancel_execution]
     cancel_execution = WorkflowsCancelExecutionOperator(
         task_id="cancel_execution",
         location=LOCATION,
         project_id=PROJECT_ID,
         workflow_id=SLEEP_WORKFLOW_ID,
-        execution_id=create_execution_id,
+        execution_id=cancel_execution_id,
     )
     # [END how_to_cancel_execution]
 
-    create_workflow >> update_workflows >> [get_workflow, list_workflows]
-    update_workflows >> [create_execution, create_execution_for_cancel]
+    delete_workflow_for_cancel = WorkflowsDeleteWorkflowOperator(
+        task_id="delete_workflow_for_cancel",
+        location=LOCATION,
+        project_id=PROJECT_ID,
+        workflow_id=SLEEP_WORKFLOW_ID,
+    )
+    delete_workflow_for_cancel.trigger_rule = TriggerRule.ALL_DONE
```

Comment:
If the trigger rule is not inside # START / # END documentation markers we can set it directly in operator constructor:
```
    delete_workflow_for_cancel = WorkflowsDeleteWorkflowOperator(
        task_id="delete_workflow_for_cancel",
        location=LOCATION,
        project_id=PROJECT_ID,
        workflow_id=SLEEP_WORKFLOW_ID,
        trigger_rule=TriggerRule.ALL_DONE,
    )
```